src/resources/ha-date-picker-style.js

```diff
@@ -1,67 +1,30 @@
 const documentContainer = document.createElement("template");
```

Comment:
I only added this file back in the day because vaadin didn't properly support ES modules. Maybe we can drop this?